blackmarket/Extensions/ViewControllerExtension.swift

```diff
@@ -27,9 +27,9 @@ extension UIViewController {
     }
     
     guard let viewController =
-      storyboard?.instantiateViewController(withIdentifier: identifier) else {
-        assert(false, "No view controller found with that identifier")
-        return
+            storyboard?.instantiateViewController(withIdentifier: identifier) else {
+      assert(false, "No view controller found with that identifier")
```

Comment:
are you using Storyboard actually in the project? do you consider to start the training with programmatic views (if you're planning to start with UIKit) or directly SwiftUI? IMO Storyboard will be not longer used in futures projects anymore

Reply:
I am using UIKit and the idea is to train with SwiftUI afterwards, I brought the change of the ios base project, I don't use this. So I took it out. Thanks! Maybe we can get it out of the ios base project as well.